HelpSource/Guides/standalones.schelp

```diff
@@ -3,11 +3,11 @@ SUMMARY:: A guide to creating SuperCollider-based applications
 CATEGORIES:: Platform>OSX, Tutorials
```

Comment:
I tried adding this category to a random .schelp file and it seems to have compiled no problem. I think Quarks tend to add their own categories as well so I see no issue here; would be great to have more input though.

<img width="371" alt="screen shot 2016-12-27 at 1 51 54 pm" src="https://cloud.githubusercontent.com/assets/15369640/21506112/afecce9a-cc3b-11e6-80dc-99323a041714.png">